_events/20240605.mdx

```diff
@@ -0,0 +1,32 @@
+---
+date: '2024-06-05T19:00:00.000Z'
```

Comment:
```suggestion
date: '2024-06-05T17:00:00.000Z'
```
If the event will start on 19pm (Europe/Rome) we need to consider our time zone (+2)